Makefile

```diff
@@ -57,12 +58,8 @@ assert-fmt:
 generate:
 	go generate $(MODULE)/...
 
-lint:
-	# We use `go list -f '{{.Dir}}' $(MODULE)/...` to skip the `vendor` folder.
-	go list -f '{{.Dir}}' $(MODULE)/... | grep -v pkg/nvml | xargs golint -set_exit_status
-
-vet:
-	go vet $(MODULE)/...
+golangci-lint:
+	golangci-lint run ./...
```

Comment:
```suggestion
	golangci-lint run -c .golangci.yml ./...
```

Reply:
Is there a reason that we explicitly specify the default?

Reply:
> This patch aligns this repo with the rest of CNT team GitHub repos

We don't do this in any of the other repos. Or if we do, we're not consistent in any case. If you feel strongly about this, then go for it, but please sign off the commit.